test/ssl_old_test.c

```diff
@@ -990,7 +992,7 @@ int main(int argc, char *argv[])
         if (strcmp(*argv, "-F") == 0) {
             fprintf(stderr,
                     "not compiled with FIPS support, so exiting without running.\n");
-            EXIT(0);
+            goto end;
```

Comment:
Oops. This inadvertently changes EXIT_SUCCESS to EXIT_FAILURE. It's not entirely clear which is more appropriate here....but I did not mean to change this, so fixup coming.

Reply:
I've noticed that but I assumed we do not use the `-F` option in our test recipes and it looks like an error-exit anyway. Perhaps it was meaningful as a test-skip kind of action at 1.0.2 FIPS module times.